ibm_catalog.json

```
@@ -0,0 +1,70 @@
+{
+    "products": [
+        {
+            "name": "terraform-ibm-resource-group",
+            "label": "Resource Group module",
+            "product_kind": "module",
+            "tags": [
+                "dev_ops",
+                "target_terraform",
+                "terraform",
+                "module"
+            ],
+            "keywords": [
+                "terraform",
+                "resource-group",
+                "group"
+            ],
+            "short_description": "Looks up or creates an IBM Cloud resource group by name",
+            "long_description": "This module returns the ID for the name of a resource group. It can also create a group from a name and return the ID.",
+            "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-resource-group/blob/main/README.md",
+            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/documentation/main/icons/GoldeneyeDeployableArchitecture.svg",
+            "features": [
+                {
+                    "title": "Create new IBM Cloud Resource Group",
```

```suggestion
                    "title": "Create an IBM Cloud resource group",
```

can't create an "old" thing, so new is redundant